backend/tests/apps/nest/graphql/nodes/user_test.py

```
@@ -0,0 +1,10 @@
+from apps.nest.graphql.nodes.user import AuthUserNode
+
+
+def test_auth_user_node_configuration():
+    """Test node configuration without DB"""
```

Ruff (0.11.9)

First line should end with a period

Add period

(D400)